astropy/uncertainty/builtin_distrs.py

```diff
@@ -0,0 +1,192 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""
+Built-in distribution-creation functions.
+"""
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import numpy as np
+
+from ..units import UnitsError
+from .core import Distribution
+
+__all__ = ['normal', 'poisson', 'uniform', 'uniform_center_width']
+
+
+def normal(center, std=None, var=None, ivar=None, n_samples=1000,
```

Comment:
I'm actually not sure about having defaults for `std` and `n_samples` here - I think both should be given by the user.

Reply:
I think we should have:

```
def normal(center, *, std=None, etc.)
```

to make sure the std etc arguments are keyword-only.